### PR TITLE
Add original occupancy classification

### PR DESCRIPTION
**Pull request by corymosiman12**
See proposals/2020/

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -14856,170 +14857,175 @@
...
-        <xs:enumeration value="Deck"/>
-        <xs:enumeration value="Courtyard"/>
-        <xs:enumeration value="Atrium"/>
-        <xs:enumeration value="Science park"/>
-        <xs:enumeration value="Other"/>
-        <xs:enumeration value="Unknown"/>
-      </xs:restriction>
-    </xs:simpleType>
   </xs:element>
+  <xs:simpleType name="OccupancyClassifications">
```
Comment (markborkum) [inline BuildingSync.xsd]: Please rename according to the naming convention for types: "OccupancyClassificationType".

**Comment by corymosiman12** [comment]
Hi @nllong sorry I think the [primary systems branch](https://github.com/BuildingSync/schema/tree/primary_systems_updates) somehow got merged into here...I only meant to try to merge the occupancy_classification stuff into develop, but it looks like it will now merge also the primary systems stuff, which I don't think is ready for primetime...

**Comment by markborkum** [comment]
@nllong Correct. Please wait to merge until Audit Template has scheduled the necessary software development resources to handle the upstream changes to BuildingSync XML schema.

**Comment by corymosiman12** [comment]
Closing this with new PR #202 to separate out only the addition of the OriginalOccupancyClassification element

**Pull request closed without merging**